copy/copy.go

```diff
@@ -191,6 +191,8 @@ type Options struct {
 	// OciDecryptConfig contains the config that can be used to decrypt an image if it is
 	// encrypted if non-nil. If nil, it does not attempt to decrypt an image.
 	OciDecryptConfig *encconfig.DecryptConfig
+	// MaxParallelDownloads indicates the maximum layers to pull at the same time
+	MaxParallelDownloads uint
```

Comment:
This should document the “built-in default” case.

Reply:
To be more precise: It should say something like “A reasonable default is used if this is left as 0” (suggesting that that’s what most callers should do; it should _not_ commit to 6 as an API guarantee.

Reply:
Fixed